src/browser/rvm/rvm_message_bus.ts

```diff
@@ -173,13 +174,27 @@ export interface Cleanup extends RvmMsgBase {
 // topic: system -----
 type systemTopic = 'system';
 type getRvmInfoAction = 'get-rvm-info';
+type sendCoreAnalyticsEvent = 'send-core-analytics-event';
```

Comment:
:neckbeard:  This is the same name as the function, not sure if that was intentional